src/Models/Xlsform.php

```diff
@@ -73,7 +73,7 @@ public function scopeOwned($builder)
                         $query->where('users.id', Auth::id());
                     } else {
                         // is the xlsform owned by a team/group other entity that the logged in user is linked to?
-                        $query->whereHas('users', function ($query) {
+                        $query->whereHas('members', function ($query) {
```

Comment:
In practice, this will be used with the Team Management package, so 'members' is correct here. Maybe in the future this can be generalised...